lms/views/admin/application_instance.py

```diff
@@ -264,13 +264,24 @@ def show_instance(self):
     def update_instance(self):
         ai = self._get_ai_or_404(**self.request.matchdict)
 
-        self.application_instance_service.update_application_instance(
-            ai,
-            lms_url=self.request.params.get("lms_url", "").strip(),
-            deployment_id=self.request.params.get("deployment_id", "").strip(),
-            developer_key=self.request.params.get("developer_key", "").strip(),
-            developer_secret=self.request.params.get("developer_secret", "").strip(),
-        )
+        try:
+            self.application_instance_service.update_application_instance(
+                ai,
+                lms_url=self.request.params.get("lms_url", "").strip(),
+                deployment_id=self.request.params.get("deployment_id", "").strip(),
+                developer_key=self.request.params.get("developer_key", "").strip(),
+                developer_secret=self.request.params.get(
+                    "developer_secret", ""
+                ).strip(),
+                organization_public_id=self.request.params.get(
+                    "org_public_id", ""
+                ).strip(),
+            )
```

Comment:
Note to self: It looks like we have all the handling to be able to update the org settings at once, but we actually use it two separate ways.

One for normal fields, and another in the _danger zone_ which only sends the single field `org_public_id`.